implementations/rust/ockam/ockam_command/src/node/delete.rs

```diff
@@ -34,7 +34,12 @@ fn run_impl(opts: CommandGlobalOpts, cmd: DeleteCommand) -> crate::Result<()> {
         delete_all_nodes(opts, cmd.force)?;
     } else {
         delete_node(&opts, &cmd.node_name, cmd.force)?;
-        println!("Deleted node '{}'", &cmd.node_name);
+        opts.shell
+            .stdout()
+            .human(format!("Deleted node_name '{}'", &cmd.node_name))
+            .machine(&cmd.node_name)
+            .json(&serde_json::json!({ "node_name": &cmd.node_name }))
```

Comment:
This is a very interesting interface.

One of the things that concerns me is how much this will balloon if we add more supported formats, and then having to go back and update existing outputs as well.

I'll need to take some time to think about this more.

Reply:
I've added a simple mechanism to select a fallback output message, hidden from the public interface.